packages/grid/src/features/standard-table/features/sorting/UseTableSortHeader.ts

```diff
@@ -20,9 +21,13 @@ export const useTableSortHeader = (columnId: string): Result => {
 
   return useMemo(() => {
     const selected = columnId === sortBy;
+    const selectedBorder = selected
+      ? "2px solid var(--lhds-color-blue-500)"
+      : "2px solid transparent";
     return {
       arrow: selected ? (desc ? "up" : "down") : undefined,
       selected,
+      selectedBorder,
```

Comment:
You already have a `selected` prop. You don't need a `selectedBorder` as well. You definitely don't need the styling in two places. Remove this.